src/components/studies/scheduler/Scheduler.tsx

```diff
@@ -56,6 +56,13 @@ const useStyles = makeStyles((theme: Theme) =>
       justifyContent: 'space-between',
       paddingRight: theme.spacing(2),
     },
+    studyStartDateContainer: {
```

Comment:
There are a lot of formatting changes in this file, but this is the only thing that I actually added.